src/components/elements/uk_account_closure_modal.tsx

```diff
@@ -0,0 +1,78 @@
+// Should move this component to a proper directory.
```

Comment:
Any idea where is the correct directory to place this component?

Reply:
I think this can be kept together in `layout` directory.

Reply:
@sean-binary  do you think we can create `modal` directory under `layout` so that future modals can go in there together?

Reply:
Yeah, can do.